### ISSUE TITLE
Error: Call to a member function bundle() on null in Drupal\views\Plugin\EntityReferenceSelection\ViewsSelection-&gt;stripAdminAndAnchorTagsFromResults()

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#8892]
## Describe the defect
```
An AJAX HTTP error occurred.
HTTP Result Code: 200
Debugging information follows.
Path: /node/add/event?ajax_form=1
StatusText: OK
ResponseText: Error: Call to a member function bundle() on null in Drupal\views\Plugin\EntityReferenceSelection\ViewsSelection-&gt;stripAdminAndAnchorTagsFromResults() (line 287 of /var/lib/tugboat/docroot/core/modules/views/src/Plugin/EntityReferenceSelection/ViewsSelection.php).

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application
  Error
      at https://pr8890-j3mfjoqe6oxvk1xqc2buutpssfm7ffsg.ci.cms.va.gov/core/misc/ajax.js?v=9.3.12:110:32
      at https://pr8890-j3mfjoqe6oxvk1xqc2buutpssfm7ffsg.ci.cms.va.gov/core/misc/ajax.js?v=9.3.12:674:3
```

## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by ndouglas** [comment]
If this happens again, I'll be sure to get a backtrace so I can actually do something 😠
